src/Microsoft.Identity.Client/HttpEvent.cs

```diff
@@ -30,35 +30,42 @@ namespace Microsoft.Identity.Client
 {
     internal class HttpEvent : EventBase
     {
+        public const string ConstHttpPath = EventNamePrefix + "http_path";
+        public const string ConstUserAgent = EventNamePrefix + "user_agent";
+        public const string ConstQueryParameters = EventNamePrefix + "query_parameters";
+        public const string ConstApiVersion = EventNamePrefix + "api_version";
+        public const string ConstResponseCode = EventNamePrefix + "response_code";
+        public const string ConstOauthErrorCode = EventNamePrefix + "oauth_error_code";
+
         public HttpEvent() : base(EventNamePrefix + "http_event") {}
 
         public string HttpPath
         {
-            set => this["http_path"] = value;
+            set => this[ConstHttpPath] = value.ToLower();
```

Comment:
We change this to lowercase now. But, on a second thought, http path are [generally believed](http://stackoverflow.com/a/7996997) to be [case-sensitive](https://tools.ietf.org/html/rfc3986#section-6.2.2.1). I would suggest to rollback the `ToLower()` treatment on this one. Same applies to the Query Parameters too.